src/dnsmasq_interface.c

```diff
@@ -800,6 +800,7 @@ void _FTL_iface(struct irec *recviface, const char *file, const int line)
 			char buffer[ADDRSTRLEN+1] = { 0 };
 			inet_ntop(AF_INET, &next_iface.addr4, buffer, ADDRSTRLEN);
 			logg("Configuration overwrites IPv4 address: %s", buffer);
+			next_iface.haveIPv4 = true;
```

Comment:
This needs to go 6 lines up I think. (thanks to @jpgpi)

Reply:
Yes, I see I pushed the wrong commit (this was from earlier testing) but the tests complained anyway so nothing would have happened :-)

Fixed now. Binaries are currently building.

Reply:
Working with the new builds even without the debug flags

Reply:
version  is now vDev-ccf8d15, tried again, appears to be working.
Thanks.